adapters/stroeerCore/stroeercore.go

```diff
@@ -0,0 +1,114 @@
+package stroeerCore
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+)
+
+type StroeerCoreAdapter struct {
+	Url string `json:"url"`
+}
+
+type StroeerRootResponse struct {
+	Bids []StroeerBidResponse `json:"bids"`
+}
+
+type StroeerBidResponse struct {
+	Id     string  `json:"id"`
+	BidId  string  `json:"bidId"`
+	Cpm    float64 `json:"cpm"`
+	Width  int64   `json:"width"`
+	Height int64   `json:"height"`
+	Ad     string  `json:"ad"`
+	CrId   string  `json:"crid"`
+}
+
+func (a *StroeerCoreAdapter) MakeBids(internalRequest *openrtb2.BidRequest, externalRequest *adapters.RequestData, response *adapters.ResponseData) (*adapters.BidderResponse, []error) {
+	if response.StatusCode != http.StatusOK {
+		return nil, []error{&errortypes.BadServerResponse{
+			Message: fmt.Sprintf("Unexpected http status code: %d.", response.StatusCode),
+		}}
+	}
+
+	var errors []error
+	stroeerResponse := StroeerRootResponse{}
+
+	if err := json.Unmarshal(response.Body, &stroeerResponse); err != nil {
+		errors = append(errors, err)
+		return nil, errors
+	}
+
+	bidderResponse := adapters.NewBidderResponseWithBidsCapacity(len(stroeerResponse.Bids))
+	bidderResponse.Currency = "EUR"
+
+	for _, bid := range stroeerResponse.Bids {
+		openRtbBid := openrtb2.Bid{
+			ID:    bid.Id,
+			ImpID: bid.BidId,
+			W:     bid.Width,
+			H:     bid.Height,
+			Price: bid.Cpm,
+			AdM:   bid.Ad,
+			CrID:  bid.CrId,
+		}
+
+		bidderResponse.Bids = append(bidderResponse.Bids, &adapters.TypedBid{
+			Bid:     &openRtbBid,
+			BidType: openrtb_ext.BidTypeBanner,
+		})
+	}
+
+	return bidderResponse, errors
+}
+
+func (a *StroeerCoreAdapter) MakeRequests(internalRequest *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	errors := make([]error, 0, len(internalRequest.Imp))
```

Comment:
Let's hope for success and save some memory/cpu by not pre-allocating the error array. Everything should still work fine if you switch this line to `var errors []error`.